<?xml version="1.0"?>
<launch>
    <include file="$(find visca_ros)/launch/camera.launch">
      <arg name="camera_name" value="zoomcamera" />
      <arg name="video_stream_provider" value="0" />
      <arg name="visualize" value="false"/>
    </include>
    <node name="visca_camera_node" pkg="visca_ros" type="visca_node" output="screen" />
</launch>
